test/functional/cypress/specs/companies/add-company-spec.js

```diff
@@ -765,6 +767,10 @@ describe('Add company form', () => {
 
     it('should display the manual entry form', () => {
       assertUSOrCanadianCompanyFormFields()
+      cy.get(selectors.companyAdd.newCompanyRecordForm.areaCanada).should(
+        'be.visible'
+      )
```

Comment:
```suggestion
      cy.get(selectors.companyAdd.newCompanyRecordForm.areaCanada).should(
        'be.visible'
      )
      cy.get(selectors.companyAdd.newCompanyRecordForm.areaUS).should(
        'not.be.visible'
      )
```
Again, just thinking out loud